### PR TITLE
Fix for HID controllers incomingData function

### PR DESCRIPTION
**Pull request by zezic**

### CODE REVIEW

**Comment by Pegasus-RPG** [comment]
What does this fix?

File: src/controllers/hid/hidcontroller.cpp [hunk 1]
```diff
@@ -20,6 +20,7 @@
 HidReader::HidReader(hid_device* device)
         : QThread(),
           m_pHidDevice(device) {
+    qRegisterMetaType<mixxx::Duration>("mixxx::Duration");
```
Comment (rryan) [inline src/controllers/hid/hidcontroller.cpp]: Please stick this in MixxxMainWindow::initialize

**Comment by zezic** [comment]
Done.

**Comment by zezic** [comment]
> What does this fix?

This makes HID controllers and incomingData function works again.

**Comment by daschuer** [comment]
LGTM

**Pull request merged**